lambdas/checkpassport/src/main/java/uk/gov/di/ipv/cri/passport/checkpassport/CheckPassportHandler.java

```diff
@@ -155,18 +157,14 @@ public APIGatewayProxyResponseEvent handleRequest(
                             userId,
                             authenticationRequest.getClientID().getValue());
             passportService.persistDcsResponse(passportCheckDao);
-            AuthorizationCode authorizationCode =
```

Comment:
I think we need to keep this returning an auth code for now otherwise that will break things. 

This will be a cleanup thing later to remove this after we have refactored the front-end to expect a "retry/finish" type response

Reply:
Added back AuthCode. This will now be returned alongside the new `result`

`{"code":{"key":"code", "value": "UUvCdKJQzWbidt_JQjSgxqJkp7wwAWpra9rgitXRzJ8"}, "result": "finish"}`